test/e2e/network_test.go

```diff
@@ -147,25 +146,19 @@ func testNorthSouthNetworking(t *testing.T) {
 	testCtx, err := NewTestContext(t)
 	require.NoError(t, err)
 
-	// Use the 0th node to test
+	// Required atleast one node to test
 	require.NotEmpty(t, gc.nodes)
-	node := gc.nodes[0]
-
-	affinity, err := getAffinityForNode(&node)
-	require.NoError(t, err, "Could not get affinity for node")
```

Comment:
Is there a reason not to use node zero?

Reply:
After removing the `node affinity` in the deployment spec, we should not get the following [error](https://issues.redhat.com/browse/WINC-523?focusedCommentId=15386757&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-15386757) . The node affinity is not required for `north-south` test and removing that allows us to deploy the windows webserver on any of the Windows Nodes during upgrade.

Reply:
Please call this out in your commit message. Ideally this should be a separate commit.

Reply:
I will add it as a separate commit